tests/app/forms/field_handlers/test_select_handler.py

```diff
@@ -54,3 +56,41 @@ class TestForm(Form):
     assert form.test_field.label.text == radio_json["label"]
     assert form.test_field.description == radio_json["guidance"]
     assert form.test_field.choices == expected_choices
+
+
+def test_get_field_with_bad_choices(value_source_resolver):
+    radio_json = {
+        "guidance": "",
```

Comment:
I can see this is a copy and paste, so understand why it is like this, but I don't think this is would validate.  Reckon you should slimline out the stuff we don't need and get rid of guidance (the bit I don't think we use like that anymore), description, q_code and validation message.

Reply:
So you mean it shouldn't validate with those parts you said to remove? Or it's invalid despite those superfluous bits.

Reply:
I can see you have already updated, it was just about reducing the amount of JSON and removing keys which aren't used anymore.